src/ui/Combobox/ComboboxValues.ts

```diff
@@ -54,13 +54,26 @@ export class ComboboxValues implements IComboboxValues {
         { id: `${this.combobox.id}-value-${index}`, className: 'coveo-combobox-value', role: 'option', tabindex: 0 },
         value.element
       ).el;
+      this.highlightCurrentQueryInSearchResults(value.element);
```

Comment:
A few months ago we adjusted the querysuggest highlighting so that the part of the suggestion _not_ in what the user typed is what gets bolded.

I think we should do the same for facet search for consistency? Or did UX recommend doing the opposite for facet search?

Reply:
Yes, UX mockup is in the Jira: https://coveord.atlassian.net/browse/JSUI-2979